[CI Failure]: `AMD: Language Models Test (Extended Pooling)` Workspace growth allocation after lock

### Name of failing test

models/language/pooling/test_token_classification.py::test_openai_privacy_filter[bfloat16-openai/privacy-filter]

### Basic information

- [ ] Flaky test
- [x] Can reproduce locally
- [ ] Caused by external libraries (e.g. bug in `transformers`)

### 🧪 Describe the failing test

models/language/pooling/test_token_classification.py::test_openai_privacy_filter[bfloat16-openai/privacy-filter] is failing on ROCm due to `AssertionError: Workspace is locked but allocation from 'modular_kernel.py:1096:_allocate_buffers' requires 15.00 MB, current size is 0.06 MB. Workspace growth is not allowed after locking.`

https://buildkite.com/vllm/ci/builds/77673/list?sid=019f5553-8716-4596-8f4c-62c6657cef31&tab=output

### 📝 History of failing test

Since transformers 5.10 upgrade

### CC List.

_No response_

--- a/tests/models/language/pooling/test_token_classification.py
+++ b/tests/models/language/pooling/test_token_classification.py
@@ -128,6 +128,10 @@ PRIVACY_FILTER_PROMPTS = [
 ]
 
 
+@pytest.mark.skipif(
+    current_platform.is_rocm(),
+    reason="Workspace growth allocation issue on ROCm. See https://github.com/vllm-project/vllm/issues/48510",
+)
 @pytest.mark.parametrize("model", ["openai/privacy-filter"])
 @pytest.mark.parametrize("dtype", ["bfloat16"])
 @torch.inference_mode

